tests/integrationtest/t/executor/executor.test

```diff
@@ -0,0 +1,676 @@
+drop table if exists t1, t2;
+create table t1 (c1 int);
+create table t2 (c1 int primary key, c2 int);
+insert into t1 values(3);
+insert into t2 values(2, 2);
+insert into t2 values(0, 0);
+delete from t1, t2 using t1 left join t2 on t1.c1 = t2.c2;
+select * from t1 order by c1;
+select * from t2 order by c1;
+## Rows in both t1 and t2 are matched, so will be deleted even if it's null.
+## NOTE: The null values are not generated by join.
+drop table if exists t1, t2;
+create table t1 (c1 int);
+create table t2 (c2 int);
+insert into t1 values(null);
+insert into t2 values(null);
+delete from t1, t2 using t1 join t2 where t1.c1 is null;
+select * from t1;
+select * from t2;
+
+# TestIssue13758
+drop table if exists t1, t2;
+create table t1 (pk int(11) primary key, a int(11) not null, b int(11), key idx_b(b), key idx_a(a));
+insert into `t1` values (1,1,0),(2,7,6),(3,2,null),(4,1,null),(5,4,5);
+create table t2 (a int);
+insert into t2 values (1),(null);
+select (select a from t1 use index(idx_a) where b >= t2.a order by a limit 1) as field from t2;
+
+# TestIssue20237
+drop table if exists t, s;
+create table t(a date, b float);
+create table s(b float);
+insert into t values(NULL,-37), ("2011-11-04",105), ("2013-03-02",-22), ("2006-07-02",-56), (NULL,124), (NULL,111), ("2018-03-03",-5);
+insert into s values(-37),(105),(-22),(-56),(124),(105),(111),(-5);
+select count(distinct t.a, t.b) from t join s on t.b= s.b;
+
+# TestToPBExpr
+drop table if exists t;
+create table t (a decimal(10,6), b decimal, index idx_b (b));
+set sql_mode = '';
+insert t values (1.1, 1.1);
+insert t values (2.4, 2.4);
+insert t values (3.3, 2.7);
+select * from t where a < 2.399999;
+select * from t where a > 1.5;
+select * from t where a <= 1.1;
+select * from t where b >= 3;
+select * from t where not (b = 1);
+select * from t where b&1 = a|1;
+select * from t where b != 2 and b <=> 3;
+select * from t where b in (3);
+select * from t where b not in (1, 2);
+drop table if exists t;
+create table t (a varchar(255), b int);
+insert t values ('abc123', 1);
+insert t values ('ab123', 2);
+select * from t where a like 'ab%';
+select * from t where a like 'ab_12';
+drop table if exists t;
+create table t (a int primary key);
+insert t values (1);
+insert t values (2);
+select * from t where not (a = 1);
+select * from t where not(not (a = 1));
+select * from t where not(a != 1 and a != 2);
+
+# TestDatumXAPI
+drop table if exists t;
+create table t (a decimal(10,6), b decimal, index idx_b (b));
+set sql_mode = '';
+insert t values (1.1, 1.1);
+insert t values (2.2, 2.2);
+insert t values (3.3, 2.7);
+select * from t where a > 1.5;
+select * from t where b > 1.5;
+drop table if exists t;
+create table t (a time(3), b time, index idx_a (a));
+insert t values ('11:11:11', '11:11:11');
+insert t values ('11:11:12', '11:11:12');
+insert t values ('11:11:13', '11:11:13');
+select * from t where a > '11:11:11.5';
+select * from t where b > '11:11:11.5';
```

Comment:
need restore `sql_mode = default`

Reply:
ok, added sql like
```
set @tmp=(select @@time_zone);
set @@time_zone = '+10:00';
...
set @@time_zone = @tmp;
```

Reply:
maybe `set @@time_zone = default` is enough. But this one is better.

Reply:
> maybe `set @@time_zone = default` is enough. But this one is better.

Why this one is better? IMO `set @@time_zone = default` seems easier and straightfoward.

Reply:
Okay, I'll change it to `set @@time_zone = default`

Reply:
done